components/apimgt/org.wso2.carbon.apimgt.internal.service/swagger.json

```diff
@@ -229,6 +229,13 @@
           "description" : "**Search condition**.\n label associated with the APIs\n",
```

Comment:
Why there are changes in this file?

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/11241/commits/2df6a7918cedae735bf9d306297b7fb82bbbcf71